compiler/executable/mod.rs

```diff
@@ -26,9 +28,10 @@ pub fn next_executable_id() -> u64 {
 }
 
 typedb_error! {
-    pub ExecutableCompilationError(component = "Executable compilation", prefix = "ECP") {
-        InsertExecutableCompilation(1, "Error compiling insert clause into executable.", typedb_source: Box<WriteCompilationError>),
-        DeleteExecutableCompilation(2, "Error compiling delete clause into executable.", typedb_source: Box<WriteCompilationError>),
-        FetchCompliation(3, "Error compiling fetch clause into executable.", typedb_source: FetchCompilationError),
+    pub ExecutableCompilationError(component = "Executable compiler", prefix = "ECP") {
```

Comment:
The type should be renamed as well.
```suggestion
    pub ExecutableCompilerError(component = "Executable compiler", prefix = "ECP") {
```